gdx/test/com/badlogic/gdx/math/MathUtilsTest.java

```diff
@@ -59,4 +59,20 @@ public void testRandomLong() {
 			assertTrue((r = MathUtils.random(-8L, -1L)) <= -1L && r >= -8L);
 		}
 	}
+
+	@Test
+	public void testSinDeg() {
+		assertEquals(0f, MathUtils.sinDeg(0f), Float.MIN_NORMAL);
+		assertEquals(1f, MathUtils.sinDeg(90f), Float.MIN_NORMAL);
+		assertEquals(0f, MathUtils.sinDeg(180f), Float.MIN_NORMAL);
+		assertEquals(-1f, MathUtils.sinDeg(270f), Float.MIN_NORMAL);
+	}
+
+	@Test
+	public void testCosDeg() {
+		assertEquals(1f, MathUtils.cosDeg(0f), Float.MIN_NORMAL);
+		assertEquals(0f, MathUtils.cosDeg(90f), Float.MIN_NORMAL);
+		assertEquals(-1f, MathUtils.cosDeg(180f), Float.MIN_NORMAL);
+		assertEquals(0f, MathUtils.cosDeg(270f), Float.MIN_NORMAL);
+	}

```

Comment:
Don't we expect `delta` of 0 instead of `Float.MIN_NORMAL` now?

Reply:
0.0f would be fine here, though `assertEquals(double, double)` (which compares exactly, with effectively 0 tolerance) is currently deprecated in JUnit, and using a tolerance of 0 just sidesteps that deprecation without addressing it. I'm not opposed to either currently, in this case.

Reply:
In this case we expect to be exactly that value, with 0 tolerance. I think the deprecation is to encourage taking into account what's the tolerated delta on each individual assertion, not to address delta being 0 in some cases.

Reply:
OK, changed to 0.